velox/dwio/common/SelectiveStructColumnReader.cpp

```diff
@@ -359,7 +359,6 @@ void SelectiveStructColumnReaderBase::read(
   }
 
   const auto& childSpecs = scanSpec_->children();
-  VELOX_CHECK(!childSpecs.empty());
```

Comment:
why remove this check?

Reply:
'childSpecs' can be empty when there is no subfield in the top-level field like `rowType = ROW({"id", "name", "address"}, {BIGINT(), ROW({}, {}), VARCHAR()})`. This case is covered by one unit test: https://github.com/facebookincubator/velox/pull/5962/files#diff-6fab1be2e55546b78e9e9125269240c674673f03232b02195ddd5759fbbe0c19R1268-R1286.